### ISSUE TITLE
Add option for info message on empty data

### ISSUE THREAD

**Issue by apyx** [ratiw/vuetable-2#116]
[DataTables](https://datatables.net/) automatically displays an info message whenever the data to display is an empty list.
Example:

![Example](https://cloud.githubusercontent.com/assets/26137734/25272782/7975ac9e-2689-11e7-8494-bdc574a97289.png)

It would be nice to have something similar for Vuetable (possibly in the form of an optional prop in ``Vuetable`` to maintain compatibility); right now what happens is you get a table with an empty ``tbody``, which I feel doesn't convey much.

Now:

![Example](https://cloud.githubusercontent.com/assets/26137734/25272963/2389f730-268a-11e7-8170-2880daddf6d5.png)

After (example):

![Example](https://cloud.githubusercontent.com/assets/26137734/25272974/2c0edf88-268a-11e7-905f-5046ef8bed03.png)

**Comment by ratiw** [comment]
@apyx It's [here](https://github.com/ratiw/vuetable-2/blob/develop/src/components/VuetablePaginationInfoMixin.vue#L21)

**Comment by apyx** [comment]
Correct, right now I'm using a CSS hack to make the pagination info component look like the table's body when the data is empty.
Just thought it'd be also nice to have an option to display a message in the table's body.

**Comment by ratiw** [comment]
@apyx Don't have time right now. But if you could send a PR, I'll consider it.
